spec/spec_helper.rb

```diff
@@ -1,18 +1,18 @@
+# frozen_string_literal: true
```

Comment:
This should be fixed in our modulesync.

Reply:
yeah, this was just a quick demo PR.  I spent no real time verifying the autocorrected changes. Some were generated with `rubocop -a` and others with `rubocop -A`. I should have really split each cop into a separate commit to make it more obvious.